### PR TITLE
translate phishing workshop to Brazilian Portuguese

### PR DESCRIPTION
**Pull request by aminueza**
This PR translates:

- Phishing workshop to pt-br

### CODE REVIEW

**Comment by aminueza** [comment]
@felladrin another workshop to check. Thank you for helping us

**Review by ozhang22 — APPROVED** [review]
Looks good. We don't have to do this now, but ideally, we should also translate the text in the images to Portuguese as well.

File: content/brazilian-portuguese/security-phishing/other-phishing-types.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Conhecido no inglês como "_malvertising_", esse tipo de golpe utiliza anúncios publicitários maliciosos que contém scripts ativos e projetados para baixar _malware_ ou forçar conteúdo indesejado em seu computador
+
+![Um exemplo de Propaganda Maliciosa](../media/malvertising.png?height=450px)
+
+Como podemos observar na imagem acima, uma propaganda maliciosa mistura-se a um site genuíno. Ele encoraja você a clicar no botão de download. E isso pode levar você a instalar um **vírus** ou **malware** em seu computador.
+
+## Golpe da ferramenta de busca
+
+Alguns golpes de _phishing_ envolvem ferramentas de busca em que o usuário é direcionado para sites que oferecem produtos ou serviços de baixo custo. Assim, quando o usuário tenta comprar o produto e insere os dados do cartão de crédito, o site golpista coleta esses dados. Existem muitos sites falsos de de bancos e anúncios de produtos falsos que são usados ​​para atrair os usuários a clicar em um link malicioso.
```
Comment (felladrin) [inline content/brazilian-portuguese/security-phishing/other-phishing-types.md:28]:
```suggestion
Alguns golpes de _phishing_ envolvem ferramentas de busca em que o usuário é direcionado para sites que oferecem produtos ou serviços de baixo custo. Assim, quando o usuário tenta comprar o produto e insere os dados do cartão de crédito, o site golpista coleta esses dados. Existem muitos sites falsos de bancos e anúncios de produtos falsos que são usados ​​para atrair os usuários a clicarem em um link malicioso.
```

File: content/brazilian-portuguese/security-phishing/phishing-basics.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+description: ""
+draft: false
+weight: 2
+---
+
+Nossa Agente Secreta Alice explica a Bob sobre **Phishing**!!!
+
+## O que é Phishing?
+
+Assim como na pesca, usa-se uma isca para pegar peixes em uma vara, **hackers** maliciosos
```
Comment (felladrin) [inline content/brazilian-portuguese/security-phishing/phishing-basics.md:12]:
```suggestion
Assim como na pesca usa-se uma isca para pegar peixes em uma vara, **hackers** maliciosos
```

File: content/brazilian-portuguese/security-phishing/phishing-basics.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+--->
+
+## Um olhar atento
+
+Alice ajuda Bob a entender as consequências do golpe dando uma olhada detalhada em seu e-mail!
+
+{{% notice warning %}}
+**Observação**: há momentos em que você precisa fornecer informações pessoais a um destinatário confiável.</br>
+No entanto, fornecer informações pessoais também pode levar a cair em golpes de **phishing**. Neste minicurso vamos entender como diferenciar entre um email legítimo e um email fraudulento.
```
Comment (felladrin) [inline content/brazilian-portuguese/security-phishing/phishing-basics.md:40]:
```suggestion
No entanto, fornecer informações pessoais também pode levar a cair em golpes de **phishing**. Neste minicurso vamos entender como diferenciar um email legítimo de um email fraudulento.
```

File: content/brazilian-portuguese/security-phishing/summary.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+## Conclusão
+
+Concluindo, os sinais comuns encontrados em golpes de phishing têm pelo menos uma das seguintes propriedades:
+
+1. Oferta grátis ou prêmios que chamam a atenção.
+2. Remetente incomum e desconhecido de comportamento ou caráter inesperado.
+3. Senso de urgência, pedindo que você aja rápido por causa de ofertas ou acordos por tempo limitado.
+   `Exemplo: Sua senha está prestes a expirar clique aqui para alterá-la imediatamente. `
+4. Links suspeitos: O corretor seria passar o mouse sobre o link para verificar se a URL para o qual você será direcionado é real e confiável.
```
Comment (felladrin) [inline content/brazilian-portuguese/security-phishing/summary.md:16]:
```suggestion
4. Links suspeitos: O correto seria passar o mouse sobre o link para verificar se a URL para a qual você será direcionado é real e confiável.
```

File: content/brazilian-portuguese/security-phishing/summary.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+   `Exemplo: mail@g00gle.com em vez de mail@google.com`
+6. Anexos de e-mail com extensões ou nome de arquivos aleatórios.
+
+## Solução
+
+Na atividade "**Você acha isso suspeito?**" os seguintes e-mails e textos foram possíveis tentativas de **phishing**:
+
+1. O e-mail das contas do spotify.
+2. O texto da Amazon para Airpods grátis.
+3. Um e-mail com uma mensagem de voz.
```
Comment (felladrin) [inline content/brazilian-portuguese/security-phishing/summary.md:27]:
```suggestion
1. O e-mail das contas do Spotify.
2. O texto da Amazon para Airpods grátis.
3. O e-mail com uma mensagem de voz.
```

**Review by felladrin — COMMENTED** [review]
Thanks for translating it!
I reviewed the files and left a few suggestions!

**Review by felladrin — APPROVED** [review]

**Pull request merged**